prisma/schema.prisma

```diff
@@ -15,16 +15,15 @@ model order {
   date          DateTime   @default(now()) @db.Timestamp(0)
   stockPrice    Float      @map("stock_price")
   stockShares   Float      @map("stock_shares")
-
   @@index([fkWalletId], map: "fk_account_id_idx")
   @@index([fkStockTicker], map: "fk_order_stock_stock_ticker_idx")
 }
 
 model player {
   playerId Int      @id @default(autoincrement()) @map("player_id")
   name     String   @db.VarChar(100)
-  email    String   @db.VarChar(45)
-  phone    String   @db.VarChar(45)
+  email    String   @unique(map: "email_UNIQUE") @db.VarChar(45)
```

Comment:
Where is this syntax from, I can't find it in the documentation?
@unique(map: "email_UNIQUE")
Wondering specifically about the "email_UNIQUE"